create_framework/http_kernel_httpkernel_class.rst

```diff
@@ -66,22 +66,22 @@ framework: it matches the incoming request and populates the request
 attributes with route parameters.
 
 Our code is now much more concise and surprisingly more robust and more
-powerful than ever. For instance, use the built-in ``ExceptionListener`` to
+powerful than ever. For instance, use the built-in ``ErrorListener`` to
 make your error management configurable::
 
     $errorHandler = function (Symfony\Component\ErrorHandler\Exception\FlattenException $exception) {
         $msg = 'Something went wrong! ('.$exception->getMessage().')';
 
         return new Response($msg, $exception->getStatusCode());
     };
-    $dispatcher->addSubscriber(new HttpKernel\EventListener\ExceptionListener($errorHandler));
+    $dispatcher->addSubscriber(new HttpKernel\EventListener\ErrorListener($errorHandler));
 
-``ExceptionListener`` gives you a ``FlattenException`` instance instead of the
+``ErrorListener`` gives you a ``FlattenException`` instance instead of the
```

Comment:
now it gives you a `FlattenException` instance if it's typehined as such in your controller or if the exception argument is not typed at all, otherwise it will give you a `\Throwable` instance.